.github/workflows/R-CMD-check.yaml

```diff
@@ -64,6 +64,7 @@ jobs:
         if: runner.os == 'macOS'
         continue-on-error: true
         run: |
+          Rscript -e "install.packages('sf')"
```

Comment:
@rachelannieswain  I think this line should be placed after the brew install lines as sf will need those libraries  before it can be installed